mpvs.lua

```diff
@@ -0,0 +1,290 @@
+--[[
+mpv scopes
+
+Copyright 2017 Paul B Mahol
+License: public domain
+Source: https://paste.ubuntu.com/25318437/
```

Comment:
This is changed a lot since @richardpl's post, so it may make sense to add a subsequent copyright notice for 2018 with the same license suggested.